### PR TITLE
Add func to check if cluster is SNO

### PR DESCRIPTION
**Pull request by kasturinarra**
Add func to check if cluster is SNO

### CODE REVIEW

**Comment by kasturinarra** [comment]
@geliu2016 could you please help review the function here ?

**Comment by geliu2016** [comment]
LGTM,thanks

File: features/step_definitions/env.rb [hunk 1]
```diff
@@ -76,3 +76,13 @@
   end
 end
 
+Given(/^I have a single node cluster$/) do
```
Comment (liangxia) [inline features/step_definitions/env.rb:79]: Perhaps `Given(/^the cluster is Single Node OpenShift$/)`

The current name make me think it's another cluster beside the current using one.

Comment (kasturinarra) [reply features/step_definitions/env.rb]: Have updated, could you please help check ? Thanks !!

**Pull request merged**